starport.ci.yml

```diff
@@ -1,34 +1,44 @@
+version: 1
 build:
```

Comment:
are we still using this file?

Reply:
Yes , we are using for ignite-cli to start the chains for testing the ibc transactions